### ISSUE TITLE
Strange "stuck on wall" bug involving banana peel collision

### ISSUE THREAD

**Issue by TimrodDX** [space-wizards/space-station-14#3779]
https://cdn.discordapp.com/attachments/790656972801572905/827020249324060752/unknown.png
https://cdn.discordapp.com/attachments/790656972801572905/827020149403418684/20210331202014_1.jpg

The above two are from one round.

https://cdn.discordapp.com/attachments/790656972801572905/826865004808503296/unknown.png
https://cdn.discordapp.com/attachments/790656972801572905/826865233988550726/unknown.png
https://cdn.discordapp.com/attachments/790656972801572905/826865264595173447/unknown.png

These three are from an earlier round where the same thing happened. I'm not sure how to reproduce or what the clown was doing to make the banana collision boxes stick to him (they'd move when he did). The collision boxes match the number of banana peels he was currently carrying in his backpack at the time.

![image](https://user-images.githubusercontent.com/7907891/113239830-81202e00-9279-11eb-87ae-5e453c011955.png)

Being buckled to a chair will fix the glitch and allow you to move again, but it's definitely possible to get stuck again. I don't know yet whether removing the banana peels from your backpack reduces or eliminates the trailing bounding boxes, and I don't have time to test for reproduction right now but will come back later to do so.

**Comment by TimrodDX** [comment]
I did some testing, and here's what I found out. This appears to be a problem of the collision box of banana peels not being removed when they're inserted into a container.  This is what causes people to get stuck. Here's how to reproduce:

- Spawn (or make) a bunch of banana peels.
- Fill your backpack with them.
- Turn on showbb and notice there's a dozen or more banana peel collision boxes over yours.
- Attempt to go through a south-facing airlock (the one in botany works)
- You'll get stuck if the banana peel hitboxes touch the wall at the same time you do.

Strangely, removing the banana peels from your backpack doesn't seem to fix being stuck.
